WPMedia/fusion-news-theme-blocks
# [Themes 1884 ] fix

## Description

#### Jira Ticket: [THEMES-1884](https://arcpublishing.atlassian.net/browse/THEMES-1884)

The top-table-list-block breaks on ExtraLarge and Medium display cards when a video doesn’t have a by array nested under the credits object.

## Test Steps

1. Checkout this branch `git checkout THEMES_1884-fix`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/top-table-list-block`
3. add at least one extra-large promo blocks inside top-table-list
4. content-source:- `story-feed-query`
    offset:- `3`
    query:-  `type:video`
5. make sure the article in videocenter have credits and editor empty

<img width="896" alt="Screenshot 2024-05-21 at 4 23 44 PM" src="https://github.com/WPMedia/arc-themes-blocks/assets/83021791/7320ee13-9ed3-4925-8d0f-86b600bd28c6">
<img width="1689" alt="Screenshot 2024-05-21 at 4 23 59 PM" src="https://github.com/WPMedia/arc-themes-blocks/assets/83021791/7f883260-4d42-4b2e-b09c-163fbbf65889">
<img width="1685" alt="Screenshot 2024-05-21 at 4 24 12 PM" src="https://github.com/WPMedia/arc-themes-blocks/assets/83021791/081c4bda-4442-4167-801f-0e24deb5ef69">



[THEMES-1884]: https://arcpublishing.atlassian.net/browse/THEMES-1884?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
